openedx/core/djangoapps/call_stack_manager/models.py

```diff
@@ -0,0 +1,6 @@
+"""
+Dummy models file
+
+Note -
+Django tests searches for models.py for creating test models.
+"""
```

Comment:
I think I would change this to say "django-nose loads models for tests, but only if the django app that the test is contained in has models itself. This file is empty so that the unit tests can have models"